eng/tools/specs-model/.prettierignore

```diff
@@ -0,0 +1,4 @@
+*.json
```

Comment:
Should these config files be moved up into the tools directory so that we have a shared set for all our ts tools?

Reply:
Eventually, yes.  But we will need to migrate each tool individually.